model/project_ref.go

```diff
@@ -45,6 +45,9 @@ type ProjectRef struct {
 	// the set of alert deliveries to be processed for that trigger.
 	Alerts map[string][]AlertConfig `bson:"alert_settings" json:"alert_config,omitempty"`
 
+	// TODO: remove the alerts field above and replace with a list of subscriptions
```

Comment:
Ah no, I got confused here again :(. Alerts will be removed at the end of this project when everything has been moved out of these. As we had discussed yesterday, there is no list of subscriptions, but we're just storing them in the subscriptions collection (with appropriate ownertype/owner).

Can you amend service/project.go to 
1. fetch subscriptions when loading a project, and serving this up via JSON
2. accept a list of subscriptions to upsert.
or would these fall out of scope of this change? (i need that to unblock the UI changes anyway)

Reply:
added